buffer/src/main/java/io/netty/buffer/ReadOnlyByteBufferBuf.java

```diff
@@ -208,14 +208,10 @@ public ByteBuf getBytes(int index, byte[] dst, int dstIndex, int length) {
 
     @Override
     public ByteBuf getBytes(int index, ByteBuffer dst) {
-        checkIndex(index);
-        if (dst == null) {
-            throw new NullPointerException("dst");
```

Comment:
It'd be nice if we keep the null check because the exception message will be helpful.

Reply:
@trustin I can but am curious what's special about this specific location. None of the other methods have equivalent explicit checks, which includes similar methods in this class as well as all the other `ByteBuf` impls of this same method. I would think if we have such a check here it should be done everywhere? but imho it's a bit redundant since it would be obvious which arg the NPE corresponds to anyhow so better to avoid the additional operation and verbosity.

Will defer to your decision though of course, please let me know!

Reply:
I prefer throwing a NPE whose message tells which parameter is null. However, if we don't do that in ByteBuf, it's fine as it is.